test/FunctionalTests/Microsoft.OData.Core.Tests/ODataNavigationLinkTests.cs

```diff
@@ -21,6 +22,21 @@ public class ODataNavigationLinkTests
 
         public ODataNavigationLinkTests()
         {
+            var model = new EdmModel();

```

Comment:
**Note:** The explanation for the refactor of the code in this test class in provided in the PR description